Source/DafnyCore/Dafny.atg

```diff
@@ -253,7 +255,8 @@ public Parser(Scanner/*!*/ scanner, Errors/*!*/ errors, Include include, ModuleD
 }
 
 bool IsIdentifier(int kind) {
-  return kind == _ident || kind == _least || kind == _greatest || kind == _older;
+  if (kind == _opaque) errors.Deprecated(t, "opaque is deprecated as an identifier. It will soon become a reserved word. Use a different name.");
```

Comment:
The `IsIdentifier` method is not the place to output any messages. This line should be deleted. The deprecation message should only be output in the `Ident` grammar production (as you do below).

Reply:
Right - I intended to remove this one.